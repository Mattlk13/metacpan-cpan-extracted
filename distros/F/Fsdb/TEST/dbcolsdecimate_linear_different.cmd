prog=dbcolsdecimate
args='-v -p 0.1 x -p 0.2 y'
