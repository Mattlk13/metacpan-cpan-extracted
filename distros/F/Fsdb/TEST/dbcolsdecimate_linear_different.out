#fsdb x y
# column x with range 100 and relative precision 0.1 gives threshold 10
# column y with range 100 and relative precision 0.2 gives threshold 20
0	0
1	50
11	50
12	50
22	50
23	50
33	50
34	50
44	50
45	50
50	70
50	71
50	91
50	92
50	99
100	100
# output 16 of 101 (0.1584)
#   | dbcolsdecimate -v -p 0.1 x -p 0.2 y
